frontend/uddhar-frontend/src/shared/components/Navbar.jsx

```
@@ -112,14 +158,16 @@ const Navbar = ({children}) => {
             className="self-end mb-4 mr-4 cursor-pointer"
             onClick={() => setShowMenu(false)}
           />
-          <nav className="flex flex-col gap-4">
+          <a className="flex flex-col gap-4">
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)